src/Adapter/Driver/Pdo/Pdo.php

```diff
@@ -37,6 +37,9 @@ class Pdo implements DriverInterface, DriverFeatureInterface, Profiler\ProfilerA
     /** @var array */
     protected $features = [];
 
+    /** @var Profiler\ProfilerInterface */
```

Comment:
Let's mark it `@internal` to prevent abuse?

Reply:
Done